canonical-websites/multipass.run
# Base docker image off bionic - smaller

QA
--

``` bash
./run build
docker build --tag multipass.run --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 multipass.run
```

Go to http://localhost:8099/_status/check and http://localhost:8099
